src/main/java/com/atlassian/bitbucket/jenkins/internal/client/BitbucketFilePathClient.java

```diff
@@ -42,5 +44,19 @@ public interface BitbucketFilePathClient {
      * @throws ServerErrorException       if the server failed to process the request
      * @throws BitbucketClientException   for all errors not already captured
      */
-    String getFileContent(BitbucketSCMFile scmFile);
+    String getFileContent(BitbucketSCMFile scmFile) throws IOException;
```

Comment:
If we go with the suggested changes, I don't think we need throws `IOException` here?